### ISSUE TITLE
UI navigation problem - returning to top domain

### ISSUE THREAD

**Issue by wilkowy** [RequestPolicyContinued/requestpolicy#901]
__Browser and Add-ons:__

- Browser: Firefox 48.0.2 on WinXP SP3
- RequestPolicy version: 1.0.beta13.2.2462.r2f19aa957.pre

__Steps to reproduce:__

1. Open any site with "other origins", for example: https://www.wolframalpha.com/ => https://i.postimg.cc/k4h40dGQ/rp1.png
2. Click on any of the "other" => https://i.postimg.cc/ydNxpN16/rp2.png
3. Click on the top domain.

__What happens?__

— You are stuck. To get back to main domain you have to close whole popup and reopen it.

__What should happen?__

— You should return to https://i.postimg.cc/k4h40dGQ/rp1.png view

PS. Ignore the numbers, website downloaded some content between taking both images.
BTW. Didn't the previous releases display "other origins" as X+Y, like 45+3 in rp1.png?

**Comment by myrdd** [comment]
Indeed, thanks for reporting.

**Comment by myrdd** [comment]
>  BTW. Didn't the previous releases display "other origins" as X+Y, like 45+3 in rp1.png?

as soon as all destinations of an origin are allowed (as in your case) or blocked, only the non-zero number is displayed.

**Comment by Ryuno-Ki** [comment]
@wilkowy 
> Browser: Firefox 48.0.2 on WinXP SP3

You are aware that WinXP reached its end of life?
(You don't have to justify your usage - just want to highlight that you are running a potentially insecure operating system).

**Comment by wilkowy** [comment]
@Ryuno-Ki Yes, I'm aware of that. XP is in no less secure if you know what are you doing, but if you don't then even W10 won't protect you. There are other matters it lacks of course, but considering all XP/7 were best (and without built-in spyware). Don't follow marketing bs.

The worse problem is that Github should be condemned for blocking functionality of its www in Fx48 (particular things don't work - are not clickable, unresponsive). You know what? Change user-agent to simulate 60.0 (ESR) and voila - everything works perfectly! (previously changed to 52.0 ESR, but seems they are rude enough to follow release cycle).

Anyway original problem is solved for me.
